projects/core/src/features-config/feature-toggles/config/feature-toggles.ts

```diff
@@ -308,6 +308,13 @@ export interface FeatureTogglesInterface {
    */
   a11yFocusableCarouselControls?: boolean;
 
+  /**
+   * Enables only Tab/Shift+Tab keyboard navigation in dialogs and preserved default scrolling behaviour of up/down keys.
+   * Components:
+   * - `PickupOptionDialogComponent`
+   */
```

Comment:
Most probably more dialog components will have the same change thus the list format, for now with only one component though:)